lib/csv_helper/exporter.rb

```diff
@@ -42,9 +44,20 @@ def set_class_for_export
       version.present? ? Institution.version(version.number) : Institution
     end
 
-    def format(key, value)
-      return "\"#{value}\"" if key == :ope && value.present?
+    def format_value(record, field)
+      value = field['non_hash'] ? record.send(field['column']) : record[field['column']]
+      return "\"#{value}\"" if field['column'] == :ope && record[field['column']].present?
+      value
+    end
+
+    def format_institution_value(record, field)
+      value = false_to_nil(field['non_hash'] ? record.send(field['column']) : record[field['column']])
```

Comment:
This `false_to_nil` conversion is unnecessary.
Both `false` and `nil` respond to `.present?` with `false`